app/controllers/admin/editions_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::EditionsController < Admin::BaseController
   include HistoricContentConcern
+  include Admin::EditionsHelper
```

Comment:
Just wondering why we need this added in?

Reply:
show_similar_slugs_warning?(@edition) is a helper method from that file. Maybe we should just extend the helpers to the controller though 🤔